schema.sql

```diff
@@ -0,0 +1,11 @@
+DROP DATABASE IF EXCISTS reviews
```

Comment:
```suggestion
DROP DATABASE IF EXISTS reviews
```
Small typo on EXISTS statement

Reply:
Thanks Scott i'll go ahead and fix it!